fluentlenium-core/src/test/java/org/fluentlenium/integration/PageTest.java

```diff
@@ -66,7 +66,7 @@ public void checkIsAtFailed() {
 
     @Test
     public void checkFollowLink() {
-        page.go().goToNextPage();
+        page.<IndexPage>go().goToNextPage();
```

Comment:
This is not as nice as the original, since `go` isn't assigned to a variable the compiler doesn't know what type we want it to be.